gulp/tasks/dist.js

```diff
@@ -22,6 +22,7 @@ gulp.task('zip', ['clean:zip', 'dist'], function() {
             gulp.src([
                 'dist/paper-full*.js',
                 'dist/paper-core*.js',
+                'dist/index.d.ts',
```

Comment:
@sasensi shouldn't this be called `paper.d.ts`?

Reply:
@lehni, I named it `index.d.ts` thinking that it would allow automatic binding when publishing along with node module.
See explanation [here](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html):

> Also note that if your main declaration file is named index.d.ts and lives at the root of the package (next to index.js) you do not need to mark the "types" property, though it is advisable to do so.

But we can name it differently and add a `types` entry in `package.json` file if you think this is better ?

Reply:
Well, it doesn't live at the root here, does it? It looks like we can use the `"types"` field in `package.json` to indicate this:

https://medium.com/cameron-nokes/the-30-second-guide-to-publishing-a-typescript-package-to-npm-89d93ff7bccd#a8f1

Then I'd go with `"types": "dist/paper.d.ts"`

Reply:
Yes, you're right :)